### ISSUE TITLE
User friendly explanation on "Send Bitcoin" when "Confirm Payment" blocks

### ISSUE THREAD

**Issue by vv01f** [GaloyMoney/galoy-mobile#861]
When preparing to send bitcoin there is reasons why the completion of the process is blocked, the botton to "Confirm Payment" is blocked. Those reasons are not transparent to the user and might not be easily mitigated in the process with multiple views depending on changing things going forth and back. 

One reason might be the address is recognized as an internal one? (suggestion by Kemal)
Another reason seems to be the left over amount for the fee is too low, especially when trying to empty the wallet this might make things difficult.
Also payment is blocked for low amounts … 
Maybe the address type could be a (if not currently then a future) reason…
etc.

basically the user needs to know what is going on and get advise what should be the parameters to change instead of being blocked.

**Comment by nicolasburtey** [comment]
last version should have a lot of improvement for this one. @vv01f can you let me know if you still see the same issues?

**Comment by vv01f** [comment]
aye, got already better :)

* "Amount exceeds your balance of…" is shown for the exact amount I have in the wallet – I still have to go back and edit
  + an easy option would be to deduct fee from amount (one click)
  + alternatively: directly "edit" the amount (without going "back" and "edit")

but close if that's the state you like to have

**Comment by MaxwellDG** [comment]
Closing because the main issue has been addressed. There's several different error messages that popup if the user tries presses 'Confirm payment' and there's a problem for whatever reason.

If we're gonna improve this process to allow for quick editing, then we can open a fresh issue on it.
